packages/core/testutils/src/BaseStitchIntTestHarness.ts

```diff
@@ -77,24 +79,24 @@ export default abstract class BaseStitchIntTestHarness {
       this.apps.map(app => {
         app.remove();
       })
-    ).then(() => this.adminClient.logout());
+    ).then(() => this.adminClient.adminAuth.logout());
```

Comment:
[nit] this is an example of why I think `adminAuth` should be renamed to just `auth`.